players.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE players(
+ID SERIAL,
+Last_Name varchar(50) NOT NULL,
+First_Name varchar(50) NOT NULL,
+Position varchar(3),
+Jersey_Number INTEGER UNIQUE,
+Weight INTEGER NOT NULL,
+Height INTEGER NOT NULL,
+Date_of_Birth DATE NOT NULL
```

Comment:
Great table schema. By convention though, you often see column names in lowercase and data types (except for `SERIAL`) in lowercase.
```suggestion
id SERIAL,
last_name varchar(50) NOT NULL,
first_name varchar(50) NOT NULL,
position varchar(3),
jersey_number INTEGER UNIQUE,
weight INTEGER NOT NULL,
height INTEGER NOT NULL,
date_of_birth DATE NOT NULL
```